src/pages/docs/expression-language.mdx

```diff
@@ -0,0 +1,455 @@
+## Expression Language
+
+`Expr` language - a feature in `Golem OSS`, enables users to write programs capable of manipulating worker responses, which are WebAssembly (WASM) values.
```

Comment:
This makes me think we need a name for this "glue" language so we can stop calling it "Expr language". 😆 

I would suggest [Rib](https://ceramica-ch.ch/en/glossary/potters-rib/#q=*%3A*). A rib is a tool that potters use to refine clay into a desired shape (which fits with the idea of refining the output of a function into a protocol-suitable format, like a response).

Reply:
Nice name!